programs/mango-v4/src/instructions/flash_loan.rs

```diff
@@ -343,10 +343,15 @@ pub fn flash_loan_end<'key, 'accounts, 'remaining, 'info>(
 
         let change_amount = change.amount - loan_origination_fee;
         let native_after_change = native + change_amount;
-        if bank.is_reduce_only() {
+        if bank.are_deposits_reduce_only() {
```

Comment:
minor: for whatever reason a function starting with `are_` feels unusual, weird

Reply:
how would you name it? I use is for singular and are for plural usually